recipes/micromamba/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: micromamba
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/mamba/archive/{{ version }}.tar.gz
+  sha256: 2c1029d350be219c682ea803004cadd4dc40c9a5029784769684530c9a17c0ef
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Comment:
I have linux envy now! 😢

Reply:
It also already works on OS X 🙂 we’ll figure out windows soon :)

Reply:
If you want to help with static build of the dependencies... I think that’s required. Libsolv and krb5 are probably missing

Reply:
No big rush on my side - I can test it out on WSL2!

Reply:
> If you want to help with static build of the dependencies..

I doubt I have the expertise but more importantly, I wouldn't have time in the near future. If that changes and it's still a blocker I would be happy to give it a crack